db/queries/assessment_records/queries.py

```diff
@@ -830,7 +871,9 @@ def get_export_data(
     output = {}
     if report_type == "OUTPUT_TRACKER":
         output = get_assessment_records_by_round_id(
-            round_id, list_of_fields[report_type].get("score_fields", None)
+            round_id,
```

Comment:
can we add a TODO on 854 to remove that custom code after the assessment closes please

Reply:
Added comment